docs/src/surfaces/titleBar/titleBar.jsx

```diff
@@ -139,12 +141,13 @@ export default class ViewsTitleBar extends React.Component {
                     </Highlighter>
 
                     {/* Sub Title */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
+                    <Heading variant="h2">
                         Sub Title
-                        <Header.Subheader>
-                            Title Bar's can have a sub title.
-                        </Header.Subheader>
-                    </Header>
+                    </Heading>
+
+                    <Typography variant="body1">
+                        Title Bar's can have a sub title.
```

Comment:
I don't think we need the apostrophe here.
> Title Bars can have a sub title.

Reply:
Fixed.

Reply:
Looks good.  Resolving.